content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -48,15 +48,15 @@ container and no startup probe defined, or as a result of its `startupProbe` suc
 
 Here's an example of a Deployment with two containers, one of which is a sidecar:
 
-{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
+{{% code_sample language="yaml" file="application/deployment-sidecar-legacy.yaml" %}}
```

Comment:
I don't think this page should be showing the legacy sidecar yaml. This should be showing the new, current way with an `initContainer` entry.

Reply:
@pegasas, the current [Sidecar containers and Pod lifecycle](https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/#sidecar-containers-and-pod-lifecycle) section on this page should remain untouched, as it describes the new way of implementing sidecars, using the init containers.

To describe the legacy way of defining sidecars, create an entirely new section (possibly at the bottom of the page) named something like "Sidecars without Init Containers (Legacy Definition)". I'm sure we can come up with a better name :)